### DISCUSSION TITLE
Some fkn install doc????

### DISCUSSION THREAD

**Discussion by paciox** [hlky/stable-diffusion-webui#377] [General]
You documented any sort of stuff but forgot the basics: How to run it normally.
Make a damn guide. No, copying from colab is different and doesn't work

**Comment by telemole** [comment]
> You documented any sort of stuff but forgot the basics: How to run it normally. Make a damn guide. No, copying from colab is different and doesn't work

there are plenty of guides out there for how to install stable diffusion - maybe say thanks for an incredible interface in an incredibly short amount of time, rather than being an entitled git.

**Reply by paciox** [reply]
> > You documented any sort of stuff but forgot the basics: How to run it normally. Make a damn guide. No, copying from colab is different and doesn't work
> 
> there are plenty of guides out there for how to install stable diffusion - maybe say thanks for an incredible interface in an incredibly short amount of time, rather than being an entitled git.

Youre as witty as dumb. SD is already up and running I was referring to this one

**Comment by athu16** [comment]
This is the development repo. The main repo has an installation guide in its wiki.
https://github.com/hlky/stable-diffusion/wiki/Installation

**Reply by paciox** [reply]
Lol, thank you!
Just out of curiosity (and not in any way a personal attack to you), how one is supposed to know?

**Reply by ooofest** [reply]
@paciox Maybe actually view the front page/README of this repo, instead of pointing fingers like an entitled prince?

![image](https://user-images.githubusercontent.com/13877064/187809083-88de9f66-0716-448d-b160-e1422cee7c88.png)

**Comment by shaguru** [comment]
https://www.youtube.com/watch?v=bu2xwx-6JGk

**Reply by paciox** [reply]
Great! Thanks!

**Comment by idkartist3D** [comment]
Here's a guide for retards: https://rentry.org/GUItard
Hope you find it useful

**Reply by paciox** [reply]
Apparently, also the guide was written by a retard, under influence of bad cut drugs.
It's the most confusing and messy guide I've ever saw after the linux docs

**Reply by paciox** [reply]
It's useless that you downvote, that guide doesn't work and skips fundamental steps. I had to follow the youtube guide posted up here
